README.md

```diff
@@ -39,6 +33,8 @@ b2sdk>=0.0.0,<1.0.0
 
 ## Not released yet
 
+* Drop Python 2 and Python 3.4 support :tada:
```

Comment:
please add a link to https://www.backblaze.com/b2/docs/quick_command_line.html

Reply:
actually, I was wrong, that link is for b2cli, not for b2sdk!

Reply:
I have removed the last commit